src/pages/wallets/[walletid].js

```diff
@@ -353,13 +353,13 @@ export default function Wallet(props) {
         <Typography sx={{ mt: [2.5, 5] }} variant="h4">
           About the Wallet
         </Typography>
-        <Typography sx={{ mt: [2.5, 5] }} variant="body2">
-          <div
-            dangerouslySetInnerHTML={{
-              __html: marked.parse(wallet.about || 'NO DATA YET'),
-            }}
-          />
-        </Typography>
+        <Typography
+          sx={{ mt: [2.5, 5] }}
+          variant="body2"
+          dangerouslySetInnerHTML={{
+            __html: marked.parse(wallet.about || 'NO DATA YET'),
+          }}
+         />
```

Comment:
@ojafero did you test it with data of the wallet.about? I remember that's on purpose with that div, otherwise the data fails to display?

Reply:
@dadiorchen Yes, it works.
<img width="611" src="https://user-images.githubusercontent.com/53157755/208317072-5d183145-2d1b-4586-bf47-7f37caa4e256.png">